src/overlays/actors/ovl_Bg_Toki_Swd/z_bg_toki_swd.c

```diff
@@ -96,16 +96,16 @@ void func_808BAF40(BgTokiSwd* this, PlayState* play) {
     if (!GET_EVENTCHKINF(EVENTCHKINF_WATCHED_SHEIK_AFTER_MASTER_SWORD_CS) && !IS_CUTSCENE_LAYER &&
         Actor_IsFacingAndNearPlayer(&this->actor, 800.0f, 0x7530) && !Play_InCsMode(play)) {
         SET_EVENTCHKINF(EVENTCHKINF_WATCHED_SHEIK_AFTER_MASTER_SWORD_CS);
-        play->csCtx.script = D_808BBD90;
+        play->csCtx.script = gMasterSwordShowcaseCs;
```

Comment:
Feel "Showcase" isn't the right word. Maybe it should be `MasterSwordFind` as in find the Master Sword

Reply:
tbh I think its fine, it reveals it all dramatically.
Maybe "reveal" actually, lol

Reply:
As noted by mzx in discord, `EVENTCHKINF_WATCHED_SHEIK_AFTER_MASTER_SWORD_CS` is incorrect. Whatever we agree on here, it should be updated to match.

Reply:
Reveal is better yea.

Reply:
Naming this "RevealMasterSword" would make the name symmetric with the other 2 cutscene names. I don't care that much about symmetry though if you don't.